sentinel_test.go

```diff
@@ -162,6 +162,229 @@ func TestSentinelClientInit(t *testing.T) {
+							{typ: '+', string: "port"}, {typ: '+', string: "6"},
+						}},
+					}}},
+				}
+			},
+		}
+		// this connection will fail because OK slave is in 's-down' status
+		// since the next 2 connections won't update sentinel list,
+		// we update the list here.
+		sentinelWithHealthySlaveInSDown := &mockConn{
+			DoFn: func(cmd Completed) RedisResult { return RedisResult{} },
+			DoMultiFn: func(multi ...Completed) []RedisResult {
+				return []RedisResult{
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "4"},
+						}},
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "32"},
+						}},
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "31"},
+						}},
+					}}},
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "6"},
+						}},
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "8"},
+							{typ: '+', string: "s-down-time"}, {typ: '+', string: "1"},
+						}},
+					}}},
+				}
+			},
+		}
+		sentinelWithoutEligibleSlave := &mockConn{
+			DoFn: func(cmd Completed) RedisResult { return RedisResult{} },
+			DoMultiFn: func(multi ...Completed) []RedisResult {
+				return []RedisResult{
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "32"},
+						}},
+					}}},
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "8"},
+							{typ: '+', string: "s-down-time"}, {typ: '+', string: "1"},
+						}},
+					}}},
+				}
+			},
+		}
+
+		sentinelWithInvalidMapResponse := &mockConn{
+			DoFn: func(cmd Completed) RedisResult { return RedisResult{} },
+			DoMultiFn: func(multi ...Completed) []RedisResult {
+				return []RedisResult{
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						{typ: '%', values: []RedisMessage{
+							{typ: '+', string: "ip"}, {typ: '+', string: ""},
+							{typ: '+', string: "port"}, {typ: '+', string: "4"},
+						}},
+					}}},
+					{val: RedisMessage{typ: '*', values: []RedisMessage{
+						RedisMessage(*Nil),
+						//{typ: '%', values: []RedisMessage{
+						//	RedisMessage(RedisError{}),
+						//	RedisMessage(RedisError{}),
+						//}},
```

Comment:
Hi @smf8,

Just removing these comments and then I think this is good to merge. Thanks!

Reply:
Yes, they look good.